l10n_it_delivery_note_base/models/stock_delivery_note_type.py

```diff
@@ -165,28 +171,28 @@ def _prepare_dn_types_vals(self, company_id):
         This method sets values needed to search and create dn types
         """
         self._set_or_create_dn_types(
-            _("Incoming"),
+            "Incoming",
```

Comment:
Qui invece passa il nome in inglese e poi sopra lo cerca prima tradotto, come nella versione originale, poi se non lo trova lo cerca anche in inglese con la lingua `en_US` nel context.
È solo quest'ultima la modifica introdotta qui.